### ISSUE TITLE
parse error: `sh auth sess int {intf} detail`

### ISSUE THREAD

**Issue by tlofreso** [CiscoTestAutomation/genieparser#43]
Receiving error: `genie.metaparser.util.exceptions.SchemaMissingKeyError: Missing keys: `
when using `parse "show authentication sessions interface gigabitEthernet 1/12 details"` from genie cli.

IOSXE Traceback:
```
Traceback (most recent call last):
  File "src/genie/cli/commands/parser.py", line 276, in genie.cli.commands.parser.ParserCommand.parse
  File "src/genie/conf/base/device.py", line 498, in genie.conf.base.device.Device.parse
  File "src/genie/metaparser/_metaparser.py", line 302, in genie.metaparser._metaparser.MetaParser.parse
  File "src/genie/metaparser/_metaparser.py", line 284, in genie.metaparser._metaparser.MetaParser.parse
  File "src/genie/metaparser/util/schemaengine.py", line 392, in genie.metaparser.util.schemaengine.Schema.validate
genie.metaparser.util.exceptions.SchemaMissingKeyError: Missing keys: [['interfaces', 'gigabitEthernet 2/20', 'local_policies'], ['interfaces', 'gigabitEthernet 2/20', 'iif_id'], ['interfaces', 'gigabitEthernet 2/20', 'session_timeout']]
```

Log files will be sent to PYATS support email.

**Comment by tlofreso** [comment]
FYI, this happens for MAB or Dot1X interfaces.

**Comment by thaixuann** [comment]
Thanks, Anthony for letting us know the issue. 
We will look into and let you know once it fixed.

Thanks,
--Thai.

**Comment by thaixuann** [comment]
Hello @tlofreso,
We fixed the issue and genie package v19.9 has been released with the fix.
Could you please run the following command and try again to see if it works?

pip install genie --upgrade 

Let us know if there is an issue. 

Thanks,
--Thai.

**Comment by tlofreso** [comment]
FYI, this worked & we are all good now. TY!

[cid:image001.png@01D5743E.7C446870]

Tony Lofreso
Datacom Engineer
x 4308684
[cid:image002.png@01D406D5.A7BD78C0]
